src/python_testing/TC_IDM_3_2.py

```
@@ -118,44 +118,41 @@ async def test_TC_IDM_3_2(self):
                 res = await self.default_controller.WriteAttribute(
                     nodeid=self.dut_node_id,
                     attributes=[(self.endpoint, test_attribute(test_value))],
                     suppressResponse=True
                 )
-                log.info(f"Device responded to suppressResponse=True request: {res}")
-            except Exception as e:
-                # Device didn't respond (timeout or other error) - this is also acceptable
-                log.info(f"Device did not respond or encountered error: {e}")
+                got_response = True
+                log.info(f"DUT responded to suppressResponse=True request: {res}")
+            except ChipStackError:  # chipstack-ok: Using try/except to validate DUT behavior without failing the test on expected errors, assert_raises would fail the test
```

Flake8 (7.3.0)

[error] undefined name 'ChipStackError'

(F821)